<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Seamus Mullan</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>569</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>935</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>1171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>461</height>
     </rect>
    </property>
    <widget class="QWidget" name="AddAccount">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>401</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Add Account</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>220</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="0">
        <widget class="QPushButton" name="SCButton">
         <property name="text">
          <string>Snapchat</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="TTButton">
         <property name="text">
          <string>TikTok</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="FBButton">
         <property name="text">
          <string>Facebook</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="IGButton">
         <property name="text">
          <string>Instagram</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="TWButton">
         <property name="text">
          <string>Twitter</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Statistics">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>401</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Statistics</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>9</y>
        <width>171</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>941</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionGithub"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_Data"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionAccount_Logger"/>
    <addaction name="actionStatistics_View"/>
    <addaction name="actionEmbed_Gallery"/>
    <addaction name="actionDetails"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuInfo"/>
  </widget>
  <action name="actionGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionExport_Data">
   <property name="text">
    <string>Export Data</string>
   </property>
  </action>
  <action name="actionAccount_Logger">
   <property name="text">
    <string>Coroutine Manager</string>
   </property>
  </action>
  <action name="actionStatistics_View">
   <property name="text">
    <string>Statistics View</string>
   </property>
  </action>
  <action name="actionEmbed_Gallery">
   <property name="text">
    <string>Embed Gallery</string>
   </property>
  </action>
  <action name="actionDetails">
   <property name="text">
    <string>Details</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
